src/main/java/org/springframework/samples/petclinic/vet/Vet.java

```diff
@@ -76,4 +73,17 @@ public void addSpecialty(Specialty specialty) {
         getSpecialtiesInternal().add(specialty);
     }
 
+    @Override
```

Comment:
I know that it can be a good idea to implement `equals()` and `hashCode()` for entities. Not sure these implementations are good enough to show best practice. Are they actually used in the app?